hub/util/compare.py

```diff
@@ -0,0 +1,55 @@
+from hub.core.tensor import Union, Tensor
+from hub.core.dataset import Dataset
+from hub.api.dataset import dataset
+from hub.client.log import logger
+from hub.util.exceptions import (
+    HashesTensorDoesNotExistError,
+)
+from hub.constants import HASHES_TENSOR_FOLDER
+import os, glob
+
+
+def jaccard_similarity(list_1, list_2):
+    """Calculated the Jaccard similarity score (also known as Intersection over Union) for the two
+       lists being compared
+
+    Args:
+        list_1: First list being compared.
+        list_2: Second list being comared.
+
+    Returns:
+        A similarity score that ranges from 0.0 to 1.0. The higher the score, the more similar the lists being compared.
+    """
+    intersection = len(list(set(list_1).intersection(list_2)))
+    union = (len(set(list_1)) + len(set(list_2))) - intersection
+    return float(intersection) / union
+
+
+def compare(dataset_1: Dataset, dataset_2: Dataset) -> int:
```

Comment:
can we move all these functions into the `hash` util file?

also this function should be called `compare_hashes`